Jenkinsfile

```diff
@@ -122,6 +122,7 @@ pipeline {
         NPM_CONFIG_USERCONFIG = '.npm/rc'
         SPAWN_WRAP_SHIM_ROOT = '.npm'
         GITHUB_PACKAGES_TOKEN = credentials('github-api-token')
+        NPM_PUBLISH_TOKEN = credentials('npm-publish-token')
```

Comment:
you'll also need to add this to `versioner()` below like:

```
, NPM_PUBLISH_TOKEN: "${NPM_PUBLISH_TOKEN}"
```

Reply:
ahhhh, good catch